testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -218,6 +232,39 @@ impl Experiment for PerformanceBenchmark {
     }
 }
 
+impl PerformanceBenchmark {
+    fn maybe_start_backup(&self) -> Result<Option<JoinHandle<()>>> {
+        if !self.backup {
+            return Ok(None);
+        }
+
+        let mut rng = ThreadRng::default();
+        let validator = self
+            .up_validators
+            .choose(&mut rng)
+            .ok_or_else(|| anyhow!("No up validator."))?
+            .clone();
+
+        const COMMAND: &str = "while true; do \
+            /opt/libra/bin/db-backup one-shot backup \
+            --max-chunk-size 1073741824 --backup-service-port 7777 \
+            --state-version $(/opt/libra/bin/db-backup one-shot query  --backup-service-port 7777 --latest-version | sed -n 's/latest-version: //p') \
+            local-fs --dir $(mktemp -d -t libra_backup_XXXXXXXX); \
+            done";
+
+        Ok(Some(tokio::spawn(async move {
```

Comment:
We seem not to get any signal in test on whether backup failed.
So if it start failing in nightly or in commit check, we will never know.

We can, for example, peek join handle in before dropping it to see if async fn completed

Reply:
@andll  even if it keeps looping, there can still be issues. I would output backup throughput from the backup service going forward and monitor that.

Reply:
I see. Do you mean watching for prometheus metric, or something else?

Reply:
(You can also just pull metrics endpoint from the instance.debug_client())